infrastructure/terraform/main.tf

```diff
@@ -266,45 +269,81 @@ resource "google_dns_record_set" "terracotta_recordset" {
 ################
 
 # Now the helm release to release all of the kubernetes manifest
-locals {
-  chart_values = {
-    grafana = {
-      admin_password = "${random_password.grafana_admin_password.result}"
-      smtp = {
-        password = "${data.google_secret_manager_secret_version.sheerwater_sendgrid_api_key.secret_data}"
-      }
-      pv = {
-        name = "${google_compute_disk.sheerwater_benchmarking_grafana.name}"
-        size = "${google_compute_disk.sheerwater_benchmarking_grafana.size}"
-      }
-      domain_name = "${trimsuffix(google_dns_record_set.grafana_recordset.name, ".")}"
-      ip_name = "${google_compute_global_address.grafana_address.name}"
-    }
-    postgres = {
-      pv = {
-        name = "${google_compute_disk.sheerwater_benchmarking_db_ssd.name}"
-        size = "${google_compute_disk.sheerwater_benchmarking_db_ssd.size}"
-      }
-      admin_password = "${random_password.db_admin_password.result}"
-    }
-    terracotta = {
-      sql_user = "read"
-      sql_password = "${random_password.postgres_read_password.result}"
-      domain_name = "${trimsuffix(google_dns_record_set.terracotta_recordset.name, ".")}"
-      ip_name = "${google_compute_global_address.terracotta_address.name}"
-      pv = {
-        name = "${google_compute_disk.sheerwater_benchmarking_terracotta.name}"
-        size = "${google_compute_disk.sheerwater_benchmarking_terracotta.size}"
-      }
-    }
-  }
-}
-
-
-# Now monitor the deployed resources with uptime robot
 resource "helm_release" "sheerwater_benchmarking" {
   name = "sheerwater-benchmarking"
   chart = "../helm/sheerwater-benchmarking"
   namespace = "sheerwater-benchmarking"
-  values = [yamlencode(local.chart_values)]
+
+  # load the default values from the helm chart
+  values = [
+    file("../helm/sheerwater-benchmarking/values.yaml")
```

Comment:
I don't think this is necessary - I'm pretty sure values merges with the default already

Reply:
I've now been able to test that loading it here allows TF to pick up changes to the values outside of TF and force a helm deployment without bumping the chart version. Helm would still do the merge outside of TF without this but we would need to bump the version for TF to pick up changes.